ci: Build periodic jobs with DEBUG=1

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #67192

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

seemethere committed:
ci: Build periodic jobs with DEBUG=1

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

[ghstack-poisoned]

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

[ghstack-poisoned]

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

[ghstack-poisoned]

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

[ghstack-poisoned]

cpuhrsch commented:
Looks like the debug test finishes in 1h20m, if I'm not misunderstanding [the logs](https://github.com/pytorch/pytorch/runs/4013592763?check_suite_focus=true). However, this includes a failure due to the distributed tests. Could we rebase this and try again?

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31902447).

seemethere committed:
Update on "ci: Build periodic jobs with DEBUG=1"


Signed-off-by: Eli Uriegas <eliuriegasfb.com>

Differential Revision: [D31902447](https://our.internmc.facebook.com/intern/diff/D31902447)

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31902447).
